build.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cat decoupled.js decoupled/* > decoupled-browser.js
```

Comment:
Why it's not in `package.json`?

Reply:
It can be. I just did it on the fly in my way. I will change that.

Reply:
Done